frontend/app/calculators/transportation/trip-results.js

```diff
@@ -131,6 +131,8 @@ export default class TransportationTripResultsCalculator extends EmberObject {
         total: truckTotal,
       };
 
+      // Add 'auto' and 'taxi' splits to each temporal split,
+      // if those two modes are listed in `this.modes`
```

Comment:
@godfreyyeung thanks for adding this comment - looking at the logic here, this code is pretty hard to read.  I think this is an example of code that's pretty "imperative."   Just mentioning as when we think about refactoring, writing declarative code will make it much easier to read and understand.
https://tylermcginnis.com/imperative-vs-declarative-programming/